wherobots/db/cursor.py

```diff
@@ -75,10 +75,16 @@ def fetchmany(self, size: int = None):
         return results
 
     def fetchall(self):
-        return self.__get_results()[self.__current_row :]
+        return self.__get_results()[self.__current_row:]
+
+    def close(self):
+        """Close the cursor."""
 
     def __iter__(self):
         return self
 
     def __next__(self):
         raise StopIteration
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
```

Comment:
Don't you need to also add the `__enter__` method (returning `self`) for the context manager to work?

Reply:
fixed, thanks